contracts/tokenomics/maker/src/contract.rs

```diff
@@ -7,17 +7,20 @@ use astroport::maker::{BalancesResponse, ConfigResponse, ExecuteMsg, Instantiate
 use astroport::pair::{Cw20HookMsg, QueryMsg as PairQueryMsg};
 use astroport::querier::query_pair_info;
 use cosmwasm_std::{
-    attr, entry_point, to_binary, Addr, Attribute, Binary, Coin, Deps, DepsMut, Env, MessageInfo,
-    QueryRequest, Reply, ReplyOn, Response, StdError, StdResult, SubMsg, Uint128, Uint64, WasmMsg,
-    WasmQuery,
+    attr, entry_point, to_binary, Addr, Attribute, Binary, Coin, Decimal, Deps, DepsMut, Env,
+    MessageInfo, QueryRequest, Reply, ReplyOn, Response, StdError, StdResult, SubMsg, Uint128,
+    Uint64, WasmMsg, WasmQuery,
 };
 use cw2::set_contract_version;
 use std::collections::HashMap;
+use std::str::FromStr;
 
 // version info for migration info
 const CONTRACT_NAME: &str = "astroport-maker";
 const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
 
+const DEFAULT_MAX_SPREAD: &str = "0.01";
```

Comment:
May be it'd be better to use percents instead of str

Reply:
u64, then use Decimal::percent() to convert const DEFAULT_MAX_SPREAD to Decimal. 
"0.01" = 1%